ltu/engine/cli.py

```diff
@@ -63,7 +63,10 @@ def ltuengine_process_dir(actions, application_key, input_dir, host=None, nb_thr
         all_threads[i] = int(all_threads[i])
     # other parameters
     offset = int(offset)
-    files = glob.glob("{}/*".format(input_dir))
+    files = []
+    for dirpath, dnames, fnames in os.walk(input_dir):
```

Comment:
dnames doesn't seem to be used, you should use _ instead to show that it is not used (and to prevent linters from reporting an unused variable)